R/retro.R

```diff
@@ -97,20 +106,20 @@ retro <- function(dir = getwd(), masterdir = lifecycle::deprecated(),
   if (length(startfile) == 0) {
     stop("No starter.ss file found in ", olddir)
   }
-
+  
   # read original starter (later written to each folder)
   startfile <- file.path(olddir, startfile)
   starter <- SS_readstarter(startfile, verbose = FALSE)
   subdirnames <- paste0(subdirstart, years)
-
+  
   # check for executable
   check_exe(exe = exe, dir = olddir, verbose = verbose)
-
+  
   # loop over retrospective years
-  for (iyr in seq_along(years)) {
+  furrr::future_walk(seq_along(years), function(iyr) {
```

Comment:
I believe that you could just use:
```
furrr:::future_walk(years, function(iyr){}
```
instead of having seq_along(years)
You would also have to change
```
starter[["retro_yr"]] <- years[iyr]
```
to 
```
starter[["retro_yr"]] <- iyr
```
on line 135

Reply:
Would also have to change line 120 to be something like:
```
newdir_iyr <- file.path(newdir, subdirnames[which(years==iyr)])
```
if not using seq_along(years)

Reply:
Thanks! Honestly, I did not actually write this code. For `retro()` I basically just switched out the `for` loop for `future_walk()`. It seems like it is kind of a six to one, half a dozen to the other situation?

Reply:
It's probably clunkier code, but I like `iyr` as an index and `years` as a vector so will just leave as-is for now.